### DISCUSSION TITLE
Is the subpixel rendering superior to GNOME?

### DISCUSSION THREAD

**Discussion by kirawi** [pop-os/cosmic-text#140] [General]
At least in PopOS, I noticed that there is a lot of color fringing on text on a 4k monitor despite using GNOME tweaks to adjust it. Will this improve this to be more like Windows/MacOS?

**Comment by kirawi** [comment]
My experience was caused by my display resetting to 100 sharpness when I switched from Windows to PopOS. Lowering it fixed it,

**Comment by jackpot51** [comment]
GNOME is a desktop environment, not a text rendering library. Applications on GNOME usually use GTK3 or GTK4. The text rendering is probably charactarized by a combination of some system libraries like cairo, pango, freetype, and the GTK version. Given the possible differences of these libraries, I can't say if cosmic-text will be any different, or how different it would be, from the typical GTK application's text rendering.
